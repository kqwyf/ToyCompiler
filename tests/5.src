0123 321 9 0 00 12300987
0.0 0.2 00.3 0.123 1.5 11.5 321.12300 00321.12300
0. 3. .2 .810 .0

