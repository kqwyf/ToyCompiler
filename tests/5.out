Token sequence:
0123            < CONSTANT    , 1      >
321             < CONSTANT    , 2      >
9               < CONSTANT    , 3      >
0               < CONSTANT    , 4      >
00              < CONSTANT    , 4      >
12300987        < CONSTANT    , 5      >
0.0             < CONSTANT    , 6      >
0.2             < CONSTANT    , 7      >
00.3            < CONSTANT    , 8      >
0.123           < CONSTANT    , 9      >
1.5             < CONSTANT    , 10     >
11.5            < CONSTANT    , 11     >
321.12300       < CONSTANT    , 12     >
00321.12300     < CONSTANT    , 12     >
0               < CONSTANT    , 4      >
.               < DOT         ,        >
3               < CONSTANT    , 13     >
.               < DOT         ,        >
.               < DOT         ,        >
2               < CONSTANT    , 14     >
.               < DOT         ,        >
810             < CONSTANT    , 15     >
.               < DOT         ,        >
0               < CONSTANT    , 4      >

Symbol table:
1     123
2     321
3     9
4     0
5     12300987
6     0.000000
7     0.200000
8     0.300000
9     0.123000
10    1.500000
11    11.500000
12    321.123000
13    3
14    2
15    810
